### PR TITLE
Retires explosive lance crafting to a nice farm upstate where it has plenty of room to run around

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

You can no longer craft explosive lances.

## Why It's Good For The Game

Explosive lances are unhealthy for the game in it's current iteration. Many years ago when the game was more loose and we weren't dealing with players who treat the game like competitive TTT or Town of Salem, 

They are a one shot kill weapon, which is the most powerful kind of weapon in every gamemode. @JohnFulpWillard likened it to 1f1, a concept from Town of Salem players where the town trades 1 person for 1 bad guy.

Modern ss13 design includes a significantly heavier load of antagonists that aren't fixed roundstart compared to when the e-lance went in.

When we added the e-lance, if nuke ops spawned, that was it, there was nuke ops, if you e-lanced the nuke ops and died you were dead until the next round.

Nowadays you're rolling for lone operative, blob, wizard, disease, revenant, and every other fun enjoyable antagonist role under the sun.

I can e-lance a nuke op/cultist/traitor/revolutionary/any bad guy in the game as a non-antag assistant, die, and have a good chance to roll another, way more fun antag in deadchat.

My change to make the e-lance a proper "we both die" tool didn't actually help because I didn't quite realize that to the modern SS13 player because of how we designed Dynamic and antagonists in the modern era, death is, frankly, not a punishment anymore.

It's time we admit the facts, items designed in 2015 SS13 in #12389 simply don't hold up in a healthy manner in 2022 SS13. Dying in SS13 in 2015 was a significantly different experience with different consequences than it has now, and right now "kills you when you use it" is not the same massive downside it was 7-8 years ago.

## Changelog
:cl:
del: You can no longer craft explosive lances.
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
I don't agree with the assertion that death is a bonus you're basing this off, and even if some people play this like a deathmatch, they aren't people I want to develop for

**Comment by Iamgoofball** [comment]
tl;dr for people who dont read OPs and scroll down to post
ghost roles ruined e-lances

**Comment by Shadowflame909** [comment]
But what if the antag is killing everyone and it's your last resort

Example: Wizard Round

**Comment by Maxipat112** [comment]
> ghost roles  ruined e-lances
Just make it so only observers can roll ghost antags

**Comment by GuillaumePrata** [comment]
> But what if the antag is killing everyone and it's your last resort

Make a grenade, make gripper gloves, trigger the grenade, put it in your pocket.
Reverse "get down president" the wizard.

That is to say, there are fun interactions between our system that never thought about because Elance was just too easy and efficient, removing it will push people to be creative again.

**Comment by calliophis-maculiceps** [comment]
> > ghost roles  ruined e-lances
> 
> 
> 
> Just make it so only observers can roll ghost antags
> 
> 

this is equivalent to removing fugitives from the game

**Comment by moocowswag** [comment]
Okay then ill just sabotage a cell and put it in a crafted stun prod and get the exact same results with less effort and bigger boom


The elance is a ghetto alternative to other methods of kamimazi bombing antags that dont require landing a hit in melee range.


This pr doesnt remove suicide bombing or do anything to fix the issue being presented.

**Comment by Fikou** [comment]
> Okay then ill just sabotage a cell and put it in a crafted stun prod and get the exact same results with less effort and bigger boom
> 
> The elance is a ghetto alternative to other methods of kamimazi bombing antags that dont require landing a hit in melee range.
> 
> This pr doesnt remove suicide bombing or do anything to fix the issue being presented, it only targets an iconic weapon that has alot of creativity in what you put into it (space lube espear is a funny way to clown security and give them a heart attack)
> 
> At the very least make a farm on the centcomm z level and put an espear there as an easter egg for ghosts to see.

kamimazi

**Comment by Livrah** [comment]
Does this remove C4 lances too? It requires telecrystals (or rare armory spawn), but crew's grenadelance 1clickremove is very good.

**Comment by Iamgoofball** [comment]
crew ei nath is not healthy for the game

**Comment by SeigaSeiga** [comment]
> They are a one shot kill weapon, which is the most powerful kind of weapon in every gamemode. @JohnFulpWillard likened it to 1f1, a concept from Town of Salem players where the town trades 1 person for 1 bad guy.

> crew ei nath is not healthy for the game

If this goes through based on (slight) exaggeration that'd be kinda silly. IED explosive lances put people into _near_ crit and take off limbs if used on someone who is completely unarmored. When used on say, Security, they do somewhere around 50 brute and have a good chance of not even causing wounds. The point at which explosive lances become as described is when someone trades telecrystals for a higher yield bomb or goes through the effort of making a chemical grenade. And at that point, _they have the grenade_ to do whatever with. Though, to be fair it's pretty easy to make a gib or kill-worthy explosive lance via water and potassium. Actually absurdly easy. Probably shouldn't be.

This is definitely nitpicking since getting knocked down or delimbed makes it easy for anyone else to follow up on that person, but I don't like calling something an instant kill when it isn't always that. At the very least it's context sensitive, and people aren't going to check this for themselves. Also if this is based on recent game experiences, we _are_ in the middle of a code freeze and the only yellow-icon changelog item right now mentions explosive lances [oops.], which isn't exactly ordinary circumstances. I'd honestly prefer if explosive lances just couldn't be made from chemical grenades instead of going away altogether, if the delivery method of the explosion is the problem and not the concept of suicide bombing in general.

**Comment by optimumtact** [comment]
I would be open to readding them as some kind of much harder to make thing, but yeah right now these are totally trash.

**Comment by castawaynont** [comment]
it's Joever

**Comment by SELFHELL** [comment]
The final victory has occurred.

**Comment by lizardqueenlexi** [comment]
Aren't we in a codebase freeze...?

**Comment by optimumtact** [comment]
i'm bending the rules for PR's  i think are important

**Comment by Shadowflame909** [comment]
admins i hope you see this and this inspires you to spawn erts more during wizard rounds

regular crewmembers are now SOL!
